src/vis_cpu/conversions.py

```diff
@@ -13,7 +14,7 @@
 try:
     from typing import Literal
 except ImportError:
-    from typing_extensions import Literal
+    from typing import Literal
```

Comment:
This just seems to repeat the line that presumably caused a TypeError in the first place.

Reply:
woops! fixed.